pkgs/tools/package-management/nix/README.md

```diff
@@ -0,0 +1,34 @@
+# Steps for Testing New Releases
+
+## Patch Releases or Updating `nixVersions.{latest,git}`
+
+Branch to merge into: **master** or **release-$nixos-version**
+
+Build the affected Nix packages and run their tests on the following platforms: **x86_64-linux**, **aarch64-linux**, **x86_64-darwin**, and **aarch64-darwin**.
+If you lack the necessary hardware for these platforms, you may need to ask others for assistance with the builds.
+Alternatively, you can request access to the Nix community builder for all platforms [here](https://github.com/NixOS/aarch64-build-box) and [here](https://nix-community.org/community-builder/).
+
+To build all dependent packages, use:
+
+```
+nix-review pr <your-pull-request>
+```
+
+And to build all important NixOS tests, run:
```

Comment:
Could be good to include the tests that detected the segfault/issues, eg:

`nixos.tests.installer`, in particular `nixos.tests.installer.simpleUefiSystemdBoot`

Reply:
(Not because of another segfault but because these do run some important nix processes, namely building and booting into a configuration)

Reply:
I would say nixos installer tests in general are important, not just `nixos.tests.installer.simpleUefiSystemdBoot`

Reply:
Yeah, installer in general

Reply:
I would say that any important tests should be included in passthru tests instead.

Reply:
Oh right it is included in the passthrough that seems better, closing this thread.

Reply:
I will include an installer test into nix in a different pull request.

Reply:
https://github.com/NixOS/nixpkgs/pull/343756